lib/dry/web/roda/templates/subapp/view__base_view.rb.tt

```diff
@@ -1,16 +1,16 @@
 # auto_register: false
 
 require "slim"
-require "dry/view/controller"
+require "dry/view"
 require "<%= config[:underscored_umbrella_name] %>/<%= config[:underscored_project_name] %>/container"
 
 module <%= config[:camel_cased_umbrella_name] %>
   module <%= config[:camel_cased_app_name] %>
     module View
-      class Controller < Dry::View::Controller
+      class BaseView < Dry::View
```

Comment:
@timriley do we still want to call it `Controller` or `BaseView`? Happy to change it back to `Controller` if desired.

Reply:
I would stay with `BaseView` Controller means something else, it's just a class with common parts extracted from other views

Reply:
@parndt could we just call this `View`? This'd mean it becomes `MyApp::MySubApp::View`. In general I prefer to avoid terms like "Base" in class names if I can.

Reply:
@timriley 	in order for things like View::Context to work this would have to be: `MyApp::MySubApp::View::View` as `Context` sits inside the `View` module namespace and I don't really like the idea of nesting `Context` inside `class View`.. what are your thoughts around that?

Reply:
I could call the Context `ViewContext`?

Reply:
ah following this lead https://github.com/dry-rb/dry-view/blob/master/lib/dry/view/context.rb#L6-L15